backend/benefit/applications/api/v1/views.py

```diff
@@ -105,9 +105,46 @@ def get_queryset(self):
         if not settings.DISABLE_AUTHENTICATION:
             if not user.is_authenticated:
                 return Application.objects.none()
-        qs = Application.objects.all().select_related("company")
+        qs = Application.objects.all().select_related("company", "employee")
         return qs
 
+    EXCLUDE_FIELDS_FROM_SIMPLE_LIST = [
+        "applicant_terms_approval",
+        "bases",
+        "attachment_requirements",
+        "applicant_terms_approval_needed",
+        "applicant_terms_in_effect",
+        "former_benefit_info",
+        "available_benefit_types",
+        "status_last_changed_at",
+        "ahjo_decision",
+        "latest_decision_comment",
+        "training_compensations",
+        "pay_subsidies",
+        "warnings",
+        "attachments",
+        "de_minimis_aid_set",
+    ]
+
+    @action(methods=["get"], detail=False, url_path="simplified_list")
+    def simplified_application_list(self, request):
+        """
+        Convenience action for the frontends that by default excludes the fields that are not normally
+        needed in application listing pages.
+        """
+        qs = self.filter_queryset(self.get_queryset())
+        context = self.get_serializer_context()
+        excludes = [
+            field
+            for field in self.EXCLUDE_FIELDS_FROM_SIMPLE_LIST
+            if field not in context.get("fields", [])
+        ]
+        context["exclude_fields"] = list(
+            set(context.get("exclude_fields", []) + excludes)
+        )
```

Comment:
I don't know if this is better or more clear but I was thinking about this in code terms and thought I'd share an alternative, you choose:
```python
fields = set(context.get("fields", []))
exclude_fields = context.get("exclude_fields", [])
extra_exclude_fields = set(self.EXCLUDE_FIELDS_FROM_SIMPLE_LIST)
context["exclude_fields"] = list(exclude_fields | (extra_exclude_fields - fields))
```